packages/visx-grid/src/types.ts

```diff
@@ -33,6 +33,8 @@ export type CommonGridProps = {
   strokeDasharray?: string;
   /** Approximate number of grid lines. Approximate due to d3 alogrithm, specify `tickValues` for precise control. */
   numTicks?: number;
+  /** An array of values that determine the number and values of the ticks. Falls back to `scale.ticks()` or `.domain()`. */
+  tickValues?: ScaleInput<Scale>[];
```

Comment:
Looks like we need import `ScaleInput` and `Scale` for this.

Reply:
I added the import for `ScaleInput`, `Scale` should be a Generic, so should not require an import.

Reply:
Nevermind, it's not generic here, let me check a little more, it might need more changes so it aligns with `AxisProps`